db/seeds/fincap.seeds.rb

```diff
@@ -585,21 +631,21 @@
     Comfy::Cms::Block.new(
       identifier: 'content',
       content: <<-CONTENT
-        Thematic Reviews are short overviews of key findings from multiple UK-based
-        research and ealuation reports on a particular topic. Each review will vary
-        depending on the available information, and will aim to include:
+Thematic Reviews are short overviews of key findings from multiple UK-based
```

Comment:
how come the indentation has been lost for these?

Reply:
it was adding extra spaces/indents